brigade-worker/src/k8s.ts

```diff
@@ -20,6 +22,18 @@ import { BrigadeEvent, Project } from "./events";
 const expiresInMSec = 1000 * 60 * 60 * 24 * 30;
 
 const defaultClient = kubernetes.Config.defaultClient();
+const getWatchClient = (): kubernetes.Watch => {
+  const kc = new kubernetes.KubeConfig();
+  const config = path.join(process.env.HOME, ".kube", "config");
```

Comment:
This was my biggest complaint about the watch API that got added. I don't like that we have to supply it with a config different from the one `defaultClient` loads. In my case, I sometimes override the config via `$KUBECONFIG`, so we might need to add handling for that. But that actually only impacts local dev (which I also don't do very often). So I am going to LGTM as-is.